libcnb-data/src/newtypes.rs

```diff
@@ -52,7 +52,7 @@ macro_rules! libcnb_newtype {
         $error_name:ident,
         $regex:expr
     ) => {
-        #[derive(Debug, Eq, PartialEq, ::serde::Deserialize, ::serde::Serialize)]
+        #[derive(Debug, Eq, PartialEq, ::serde::Deserialize, ::serde::Serialize, Clone)]
```

Comment:
The list of traits in this macro's rustdoc needs updating to add `Clone`

Reply:
Your attention to detail is top notch Ed. Thanks! :)